README.md

```diff
@@ -11,10 +11,10 @@ seen at [http://validator.w3.org/checklink](http://validator.w3.org/checklink).
 
 To install the distribution for command line use:
 
-   perl Makefile.PL
```

Comment:
Sorry to jump in, but fyi it's actually better to use the 3-backtick+lang+no-space format. Example:

```sh
command $env # comment
```

vs.

    command $env # comment